### PR TITLE
ENH: Wrap the class with float transforms

### PR DESCRIPTION
**Pull request by dzenanz**
We need ITK 5.4, at least below version or newer:
https://github.com/InsightSoftwareConsortium/ITK/commit/71c5c83e100a92321783b81d62564365f9557785

### CODE REVIEW

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ jobs:
     uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@d2f5d1894abf01a93cc6581c3b2f17b508170ce3
 
   python-build-workflow:
-    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@246883ef3828fc00219145aeec9efa67b9889d0b
+    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@d2f5d1894abf01a93cc6581c3b2f17b508170ce3
+    with:
+      itk-python-package-tag: v5.4rc02
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml:12]: What is the right way to get 5.4RC2?

Comment (dzenanz) [reply .github/workflows/build-test-package.yml:12]: Currently getting:
```log
CMake Error at CMakeLists.txt:31 (find_package):
-- Configuring incomplete, errors occurred!
  Could not find a configuration file for package "ITK" that is compatible
  with requested version "5.4".

  The following configuration files were considered but not accepted:

    /work/ITK-cp38-cp38-manylinux_2_28_x64/ITKConfig.cmake, version: 5.3.0



Traceback (most recent call last):
  File "/opt/python/cp38-cp38/lib/python3.8/site-packages/skbuild/setuptools_wrap.py", line 666, in setup
    env = cmkr.configure(
  File "/opt/python/cp38-cp38/lib/python3.8/site-packages/skbuild/cmaker.py", line 357, in configure
    raise SKBuildError(msg)
```

Comment (thewtex) [reply .github/workflows/build-test-package.yml:12]: Point to the commit, 3f63de3, which is the current tip of the `v5.4.0` branch: https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/tree/v5.4.0

Comment (dzenanz) [reply .github/workflows/build-test-package.yml:12]: This seems to have worked. The error is now `ImportError: cannot import name '_POLICIES' from 'auditwheel.policy' (/opt/python/cp38-cp38/lib/python3.8/site-packages/auditwheel/policy/__init__.py)`.

File: setup.py [hunk 2]
```diff
@@ -43,6 +43,6 @@
     keywords='ITK InsightToolkit',
     url=r'https://itk.org/',
     install_requires=[
-        r'itk>=5.3.0'
+        r'itk>=5.4.0'
```
Comment (thewtex) [inline setup.py:46]: 5.4rc2

**Review by thewtex — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (dzenanz) [reply setup.py:46]: We might need RC3 here, due to need for https://github.com/InsightSoftwareConsortium/ITK/pull/4444 and https://github.com/InsightSoftwareConsortium/ITKPythonPackage/issues/263.

**Review by thewtex — APPROVED** [review]

**Comment by dzenanz** [comment]
Merge if CI is green.

**Pull request merged**